templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -43,25 +74,62 @@ export default function ChatInput(
     setImageUrl(base64);
   };
 
+  const handleUploadCsvFile = async (file: File) => {
+    const content = await new Promise<string>((resolve, reject) => {
+      const reader = new FileReader();
+      reader.readAsText(file);
```

Comment:
If it's just an attachment, we'll include it every time the file is selected. The app might struggle with large CSV files, so how about we limit the file size?